R/pkg/inst/tests/testthat/test_mllib.R

```diff
@@ -454,4 +454,66 @@ test_that("spark.survreg", {
   }
 })
 
+test_that("spark.mvnormalmixEM", {
+  # R code to reproduce the result.
+  #
+  #' library(mvtnorm)
```

Comment:
if you have a block of code in comment, try doing 
```
# nolint start
# code...
# nolint end
```
 instead. switching between `#` and `#'` is a bit confusing